ports/espressif/supervisor/usb_serial_jtag.c

```diff
@@ -121,8 +121,8 @@ char usb_serial_jtag_read_char(void) {
     return c;
 }
 
-bool usb_serial_jtag_bytes_available(void) {
-    return ringbuf_num_filled(&ringbuf) > 0 || usb_serial_jtag_ll_rxfifo_data_available();
+uint32_t usb_serial_jtag_bytes_available(void) {
+    return ringbuf_num_filled(&ringbuf) || usb_serial_jtag_ll_rxfifo_data_available();
```

Comment:
This doesn't make sense to me. Shouldn't it be `+`?

Reply:
Yes, I thought the second thing was a boolean, but it is not.